graphql-mode.el

```diff
@@ -51,19 +51,27 @@
 
 (defcustom graphql-indent-level 2
   "Number of spaces for each indentation step in `graphql-mode'."
+  :tag "GraphQL"
   :type 'integer
-  :safe 'integerp)
-
+  :safe 'integerp
+  :group 'graphql)
 
-(defvar graphql-url
-  nil)
-(make-variable-buffer-local 'graphql-url)
+(defcustom graphql-url "http://localhost:8000/graphql"
```

Comment:
Can we make the initial value `nil`?  It was set up so it would prompt the user the first time it tries to execute the query.  I don't think `http://localhost:8000/graphql` is so common to deserve to be a default.

Reply:
ok, will revert `graphql-url` back to nil, by default.